cmd/soci/commands/push.go

```diff
@@ -133,7 +136,7 @@ if they are available in the snapshotter's local content store.
 			return nil
 		}
 
-		err = oraslib.CopyGraph(context.Background(), src, dst, indexDesc, options)
+		err = oraslib.CopyGraph(context.Background(), src, dst, indexDesc.Descriptor, options)
```

Comment:
what is happening here?

Reply:
We pass the index descriptor. It traverses `index.Blobs` array, pushes them to remote, before finally pushing the index itself. [Link](https://pkg.go.dev/oras.land/oras-go/v2#CopyGraph)